some lint errors

## Bug Report

Please answer these questions before submitting your issue. Thanks!

### 1. Minimal reproduce step (Required)
branch master
`make check`

### 2. What did you expect to see? (Required)
No error

### 3. What did you see instead (Required)
```shell
executor/stale_txn_test.go:1023:18: S1039: unnecessary use of fmt.Sprintf (gosimple)
                        sql:          fmt.Sprintf("select * from t"),
                                      ^
executor/stale_txn_test.go:1032:18: S1039: unnecessary use of fmt.Sprintf (gosimple)
                        sql:          fmt.Sprintf("select * from t"),
                                      ^
executor/stale_txn_test.go:1045:18: S1039: unnecessary use of fmt.Sprintf (gosimple)
                        sql:          fmt.Sprintf("select * from t"),
                                      ^
util/memory/tracker.go:78:2: S1023: redundant `return` statement (gosimple)
        return
        ^
store/copr/coprocessor_test.go:158:24: ineffectual assignment to err (ineffassign)
        _, cluster, pdClient, err := testutils.NewMockTiKV("", nil)
                              ^
```
### 4. What is your TiDB version? (Required)

branch master

--- a/executor/stale_txn_test.go
+++ b/executor/stale_txn_test.go
@@ -1020,7 +1020,7 @@ func (s *testStaleTxnSuite) TestStmtCtxStaleFlag(c *C) {
 		},
 		// assert select statement
 		{
-			sql:          fmt.Sprintf("select * from t"),
+			sql:          "select * from t",
 			hasStaleFlag: false,
 		},
 		// assert select statement in stale transaction
@@ -1029,7 +1029,7 @@ func (s *testStaleTxnSuite) TestStmtCtxStaleFlag(c *C) {
 			hasStaleFlag: false,
 		},
 		{
-			sql:          fmt.Sprintf("select * from t"),
+			sql:          "select * from t",
 			hasStaleFlag: true,
 		},
 		{
@@ -1042,12 +1042,12 @@ func (s *testStaleTxnSuite) TestStmtCtxStaleFlag(c *C) {
 			hasStaleFlag: false,
 		},
 		{
-			sql:          fmt.Sprintf("select * from t"),
+			sql:          "select * from t",
 			hasStaleFlag: true,
 		},
 		// assert select statement after consumed set transaction
 		{
-			sql:          fmt.Sprintf("select * from t"),
+			sql:          "select * from t",
 			hasStaleFlag: false,
 		},
 		// assert prepare statement with select as of statement

--- a/go.mod
+++ b/go.mod
@@ -78,6 +78,7 @@ require (
 	golang.org/x/tools v0.1.4
 	google.golang.org/grpc v1.27.1
 	gopkg.in/check.v1 v1.0.0-20201130134442-10cb98267c6c // indirect
+	honnef.co/go/tools v0.2.0 // indirect
 	modernc.org/mathutil v1.2.2 // indirect
 	sourcegraph.com/sourcegraph/appdash v0.0.0-20190731080439-ebfcffb1b5c0
 	sourcegraph.com/sourcegraph/appdash-data v0.0.0-20151005221446-73f23eafcf67

--- a/store/copr/batch_coprocessor.go
+++ b/store/copr/batch_coprocessor.go
@@ -141,13 +141,13 @@ func balanceBatchCopTask(ctx context.Context, kvStore *kvStore, originalTasks []
 				alive := false
 				resp, err := kvStore.GetTiKVClient().SendRequest(ctx, s.GetAddr(), aliveReq, tikv.ReadTimeoutMedium)
 				if err != nil {
-					logutil.BgLogger().Warn("Cannot detect store's availablity", zap.String("store address", s.GetAddr()), zap.String("err message", err.Error()))
+					logutil.BgLogger().Warn("Cannot detect store's availability", zap.String("store address", s.GetAddr()), zap.String("err message", err.Error()))
 				} else {
 					rpcResp := resp.Resp.(*mpp.IsAliveResponse)
 					if rpcResp.Available {
 						alive = true
 					} else {
-						logutil.BgLogger().Warn("Cannot detect store's availablity", zap.String("store address", s.GetAddr()))
+						logutil.BgLogger().Warn("Cannot detect store's availability", zap.String("store address", s.GetAddr()))
 					}
 				}
 				if !alive {

--- a/store/copr/coprocessor_test.go
+++ b/store/copr/coprocessor_test.go
@@ -156,6 +156,7 @@ func (s *testCoprocessorSuite) TestSplitRegionRanges(c *C) {
 	// nil --- 'g' --- 'n' --- 't' --- nil
 	// <-  0  -> <- 1 -> <- 2 -> <- 3 ->
 	_, cluster, pdClient, err := testutils.NewMockTiKV("", nil)
+	c.Assert(err, IsNil)
 	testutils.BootstrapWithMultiRegions(cluster, []byte("g"), []byte("n"), []byte("t"))
 	pdCli := &tikv.CodecPDClient{Client: pdClient}
 	cache := NewRegionCache(tikv.NewRegionCache(pdCli))

--- a/util/memory/tracker.go
+++ b/util/memory/tracker.go
@@ -95,7 +95,6 @@ func InitTracker(t *Tracker, label int, bytesLimit int64, action ActionOnExceed)
 	t.bytesSoftLimit = int64(float64(bytesLimit) * softScale)
 	t.maxConsumed = 0
 	t.isGlobal = false
-	return
 }
 
 // NewTracker creates a memory tracker.

